Library/Formula/czmq.rb

```diff
@@ -43,13 +42,9 @@ def install
 
     system "./autogen.sh" if build.head?
     system "./configure", *args
-    system "make", "install"
+    system "make"
+    system "make check"
```

Comment:
This sort of thing should be `system "make", "check"`. The comma has importance beyond just looking a little nicer.